lib/sport_ngin_aws_auditor/instance_helper.rb

```diff
@@ -84,6 +111,20 @@ def filter_instance_without_tags(instances)
       end
     end
 
+    # this gathers all RIs except the region-based RIs
+    def filter_ris_availability_zone(ris)
+      ris.reject do |ri|
```

Comment:
The common style for a short Ruby block like this where we care about the return value is like this:
```
ris.reject { |ri| ri.scope == 'Region' }
```
Same thing below.